platform/source/TARGET_CORTEX_M/TOOLCHAIN_IAR/except.S

```diff
@@ -58,7 +59,7 @@ BusFault_Handler
 UsageFault_Handler
         EXPORT  UsageFault_Handler
         MOVS    R3,#FAULT_TYPE_USAGE_FAULT
-        ; Fall into Fault_Handler
+        B       Fault_Handler
```

Comment:
Why is this required? To make Fault_Handler 4-byte aligned? But it doesn't need to be.